tools/pipeline-witness/infrastructure/bicep/appResourceGroup.bicep

```diff
@@ -1,11 +1,73 @@
 param webAppName string
+param networkSecurityGroupName string
+param vnetName string
 param appServicePlanName string
 param appStorageAccountName string
+param aspEnvironment string
 param cosmosAccountName string
 param location string
 
 var cosmosContributorRoleId = '00000000-0000-0000-0000-000000000002' // Built-in Contributor role
 
+resource networkSecurityGroup 'Microsoft.Network/networkSecurityGroups@2023-11-01' = {
+  name: networkSecurityGroupName
+  location: 'westus2'
+  properties: {
+    securityRules: []
+  }
+}
+
+resource vnet 'Microsoft.Network/virtualNetworks@2023-11-01' = {
+  name: vnetName
+  location: 'westus2'
+  properties: {
+    addressSpace: {
+      addressPrefixes: [
+        '10.8.0.0/16'
```

Comment:
Why are you adding only this prefix?

Reply:
not sure.  Was just in the import.   Should I remove it?